client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { isNumber } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+
+class ProductCategoryForm extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		siteId: PropTypes.number,
+		category: PropTypes.shape( {
+			id: PropTypes.isRequired,
+		} ),
+		editProductCategory: PropTypes.func.isRequired,
+	};
+
+	setName = e => {
+		const { siteId, category, editProductCategory } = this.props;
+		const name = e.target.value;
+		editProductCategory( siteId, category, { name } );
+	};
+
+	renderPlaceholder() {
+		const { className } = this.props;
+		return (
+			<div className={ classNames( 'product-categories__form', 'is-placeholder', className ) }>
+				<div />
+			</div>
+		);
+	}
+
+	render() {
+		const { siteId, category, translate } = this.props;
+
+		const isCategoryLoaded = category && isNumber( category.id ) ? Boolean( category.slug ) : true;
+
+		if ( ! siteId || ! category || ! isCategoryLoaded ) {
+			return this.renderPlaceholder();
+		}
+		return (
+			<div className={ classNames( 'product-categories__form', this.props.className ) }>
+				<Card>
+					<FormFieldSet>
+						<FormLabel htmlFor="name">{ translate( 'Category name' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **64** times:
`translate( 'Category Name' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-base-product-cat-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).